src/components/HomeScreenHeader.tsx

```diff
@@ -105,7 +105,7 @@ const HomeScreenHeader: React.FC<HomeScreenHeaderProps> = ({
               color={`${colorMode}.choosePlanHome`}
               medium
             >
-              {capitalizeEachWord(title === wallet.more ? common.keeperSettings : title)}
```

Comment:
This does not look good, please leave the header with the previous title and gear icon